frontend/src/styles/AppFooter.css

```diff
@@ -1,3 +1,16 @@
 .AppFooter {
     /* Styles go here */
-}
\ No newline at end of file
+    background-color: #7FC1AC;
+    position: absolute;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding: 1rem;
+    
+}
+
+svg {
+    color: #F4F5F8;
```

Comment:
Colors should be added to App.css as root variables.

Reply:
Resolved by 72b479c

Reply:
Colors added as  root variables